src/test/java/io/confluent/examples/streams/interactivequeries/kafkamusic/KafkaMusicExampleTest.java

```diff
@@ -67,9 +66,71 @@
   private KafkaStreams streams;
   private MusicPlaysRestService restProxy;
   private int appServerPort;
+  private static final List<Song> songs = Arrays.asList(new Song(1L,
```

Comment:
These sample songs are loaded from a file (thanks @ybyzek ) in `io.confluent.examples.streams.interactivequeries.kafkamusic.KafkaMusicExampleDriver`.  Maybe we could re-use the same approach here?

Reply:
Sure, @bbejeck The approach seems good and applicable. Will apply them. Thanks for pointing it out.